### DISCUSSION TITLE
Synctrain files sync to iCloud by default (fixed in v1.3)

### DISCUSSION THREAD

**Discussion by eskildsf** [pixelspark/sushitrain#3] [Q&A]
Hi

I am on the free iCloud backup plan and recently my phone stopped syncing to iCloud. I coulnd't really understand why until I checked the iCloud sync settings (buried 5 clicks deep in the Settings app) and saw that iCloud had sync enabled for the Synctrain app. I have some 30 GB of files that sync via Synctrain so of course it had filled up my iCloud storage space.

I just want to put this information here if anybody else is surprised that Synctrain files are by default also synced to iCloud.

Thanks for the app. It's amazing!

Br Eskild

**Comment by pixelspark** [comment]
Hm, is it syncing a backup including Synctrain's folders, or is it syncing the folders to iCloud Drive?

The former can be disabled in the app on a per-folder basis as of v1.3 (and should be disabled by default for newly added folders, but folders created before v1.3 will still have it enabled by default until you turn it off).

The latter would be interesting as I am not seeing that behaviour myself, but maybe I disabled something on my phone. Could you perhaps share a screenshot of the toggle you are referring to when mentioning 'disabling iCloud for Synctrain'?

**Reply by eskildsf** [reply]
Hi. Thank you for your reply.

iCloud is syncing a backup of Synctrain which includes the folders. Thank you for letting me know about the in-app per-folder setting. I'll toggle those instead.

Br Eskild
